ViconData crazyflieLocation
Setpoint setpoint
---
RateCommand controlOutput
